packages/app/src/components/challenges/OutcomeCard.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { ReactElement, useRef, useState } from 'react';
+import {
+    Text,
+    View,
+    StyleSheet,
+    TextInput,
+    TouchableOpacity,
+    KeyboardAvoidingView,
+    Platform
+} from 'react-native';
+import { Card, colors, Notch, sizes, Title } from '../UI';
+import { FormType } from './CreateChallengeScreen';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+import { ScrollView } from 'react-native-gesture-handler';
+
+type Props = {};
+
+export const OutcomeCard = ({ type, form, setForm }: any): ReactElement => {
+    const [textValue, setTextValue] = useState('');
+    const [listHeight, setListHeight] = useState(0);
+    const [warningText, setWarningText] = useState('');
+    const scrollViewRef = useRef(null);
+    const positive = type === 'positiveOptions';
```

Comment:
this can be a boolean

true === "positive"
false === "negative"

Reply:
this has been updated